docs/guide/installation/README.md

```diff
@@ -52,35 +52,66 @@ Vue.use(VueFormulate.default)
 ```
 :::
 
-### Nuxt
-It's easy to use Vue Formulate on Nuxt too. [A working example is available.](https://codesandbox.io/s/vue-formulate-test-8segh?file=/components/Attending.vue)
+### Configuration options
 
-#### plugins/vue-formulate.js
+If you need custom configuration options, you can pass a second argument with
+an object of configuration options.
 
 ```js
-import Vue from 'vue'
-import VueFormulate from '@braid/vue-formulate'
-
-Vue.use(VueFormulate)
+Vue.use(VueFormulate, options)
 ```
 
+## Nuxt
+It's easy to use Vue Formulate on Nuxt too. As of version `2.5` the easiest way
+to get start with Vue Formulate on a Nuxt project is to use the included Nuxt module.
+
+For earlier versions of Vue Formulate see this [CodeSandbox implementation](https://codesandbox.io/s/vue-formulate-test-8segh?file=/nuxt.config.js)
+with a working Vue Formulate Nuxt plugin file.
+
 #### nuxt.config.js
 
 ```js
 export default {
-  plugins: [
-     '~/plugins/vue-formulate',
+  ...
+  buildModules: [
+    '@braid/vue-formulate/dist/nuxt'
   ]
-  (...)
+  // optionally, define a path for your Vue Formulate configuration options.
+  // the default value is formulate.config.js in your Nuxt source directory
+  formulate: {
+    configPath: '~/path/to/different/formulate.config.js'
+  }
+  ...
+}
 ```
 
-## Configuration options
+### Nuxt module configuration file
 
-If you need custom configuration options, you can pass a second argument with
-an object of configuration options.
+To configure Vue Formulate with additional [plugins](/guide/plugins/),
+[locales](/guide/internationalization/), [validation rules](/guide/validation/)
+and more you can create a `formulate.config.js` file which will be imported and
+used by the Vue Formulate Nuxt module.
+
+The easiest option is to create a `formulate.config.js` file in your Nuxt project
+root and it will automatically be imported by the Nuxt module. If you wish to place
```

Comment:
```suggestion
and more you can create a `formulate.config.js` file which will be imported and
used by the Vue Formulate Nuxt module.  If you wish to place
```

Reply:
Small tweak since it sounded a bit repetitive, other than this im good to merge. great work!